resources/assets/lib/sort.tsx

```diff
@@ -40,7 +40,12 @@ export class Sort extends React.PureComponent<Props> {
           key={value}
           onClick={this.props.onSortSelected}
         >
-          {osu.trans(`sort.${value}`)}
+          {value === 'rank'
+            ? <span>
+                <i className={`fas fa-extra-mode-${currentUser.playmode || 'osu'}`} /> {osu.trans('sort.rank')}
```

Comment:
use `??` instead of `||`.

Reply:
``??`` won't work in JS, ``||`` is a proper way to do this i believe